homeassistant/components/smartthings/strings.json

```diff
@@ -530,6 +530,15 @@
       "sabbath_mode": {
         "name": "Sabbath mode"
       }
+    },
+    "water_heater": {
+      "water_heater": {
+        "state": {
+          "standard": "Standard",
+          "force": "Force",
+          "power": "Power"
```

Comment:
The only concern I have with this PR is that this list doesn't map properly to the list of pre-defined modes
```py
STATE_ECO = "eco"
STATE_ELECTRIC = "electric"
STATE_PERFORMANCE = "performance"
STATE_HIGH_DEMAND = "high_demand"
STATE_HEAT_PUMP = "heat_pump"
STATE_GAS = "gas"
```